File: Hackerrank/Operators.py
#!/bin/python3

import math
import os
import random
import re
import sys

# Complete the solve function below.
def solve(meal_cost, tip_percent, tax_percent):
    tip=meal_cost*(tip_percent/100)
    tax=meal_cost*(tax_percent/100)
    total_cost=round(meal_cost+tip+tax)
    print(total_cost)

if __name__ == '__main__':
     meal_cost = float(input())
     tip_percent = int(input())
     tax_percent = int(input())
        
solve(meal_cost, tip_percent, tax_percent)
      

File: Hackerrank/Operators.java
//https://www.hackerrank.com/challenges/30-operators/problem
//Just give the input of meal cost, tip and tax and the program will give the desired output
//Note: Tip and tax should be in integer only
import java.util.Scanner;
public class Main {
	
        static int solve(double meal_cost, int tip_percent, int tax_percent) {

            double totalCost;
            double tip;
            double tax;


            tip=(tip_percent*(meal_cost/100));
            
            tax=(tax_percent*(meal_cost/100));
            
            totalCost=meal_cost+tip+tax;
            totalCost=Math.abs(totalCost);
            
            return (int) totalCost;


        }

        static final Scanner scanner = new Scanner(System.in);

        public static void main(String[] args) {
            double meal_cost = scanner.nextDouble();
            scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");

            int tip_percent = scanner.nextInt();
            scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");

            int tax_percent = scanner.nextInt();
            scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");



            System.out.println(solve(meal_cost, tip_percent, tax_percent));

            scanner.close();
        }
}